lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -44,6 +45,10 @@
 
 
 class Test_write(tests.IrisTest):
+    # -------------------------------------------------------------------------
+    # It is not considered necessary to have integration tests for saving
```

Comment:
This refers to integration tests, but this is in iris/tests/unit . Can these still be described as integration tests?

Reply:
You're right, I hadn't considered how oddly this reads. But the tests below (e.g. test_mercator) definitely don't fit the definition of unit test in my opinion since NetCDF saving relies on a large number of steps. Could you suggest a better word than integration, which still makes it clear that these aren't unit tests?

Reply:
Taking a **very** narrow view, just omit the word "integration" : job done!

Reply:
Yeah, having read through the tests I think I agree that these are closer to integration tests than unit tests. If you want to avoid confusion, it could be worth considering just getting rid of the word "integration", though I'm more convinced now that this ought to be be fine as is.